crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -52,6 +58,13 @@ pub(crate) struct QueryLimitsChecker {
     validation_result: Mutex<Option<ValidationRes>>,
 }
 
+pub(crate) enum ParseQueryError {
+    NotFound,
+    NotU64,
```

Comment:
What's the difference between NotU64 and NotANumber? We need to distinguish between signed and unsigned numbers?

Reply:
After refactoring per @amnn's suggestion, we don't need these confusing errors anymore